pycbc/psd/__init__.py

```diff
@@ -210,7 +210,12 @@ def insert_psd_option_group(parser, output=True, include_data_options=True):
     psd_options.add_argument("--psdvar_long_segment", type=float,
                              metavar="SECONDS", help="Length of long segment "
                              "when calculating the PSD variability.")
-    psd_options.add_argument("--psdvar_overlap", type=float, metavar="SECONDS",                              help="Sample length of the PSD.")
+    psd_options.add_argument("--psdvar_short_psd_duration", type=float,
```

Comment:
I think these should be using '-' between words instead of '_' to be consistent with all the other options.

Reply:
done.